packages/keystone/lib/List/index.js

```diff
@@ -105,82 +105,96 @@ const mapNativeTypeToKeystoneType = (type, listKey, fieldPath) => {
 module.exports = class List {
   constructor(
     key,
-    config,
+    {
+      fields,
+      hooks = {},
+      mutations = [],
+      schemaDoc,
+      labelResolver,
+      labelField,
+      access,
+      adminConfig = {},
+      itemQueryName,
+      listQueryName,
+      label,
+      singular,
+      plural,
+      path,
+      ...adapterConfig
```

Comment:
There's a lot which gets bundled up into the `adapterConfig`. In a separate PR I'll try to tease out what all the various possibilities are.